README.md

```diff
@@ -60,6 +60,66 @@ jupyter lab
 
 3. Run all cells (Kernel > Restart & Run All) to reproduce the analysis and generate the visualizations.
 
+## Dependencies
+
+[Docker](https://www.docker.com/) is a container solution used to manage the software dependencies for this project. The Docker image used for this project is based on the condaforge/miniforge3:latest image. Additional dependencies are specified in the [Dockerfile](Dockerfile)
+
+### Installing Docker
+
+If you don't have Docker installed, download and install it from:
+- **Windows/Mac**: [Docker Desktop](https://www.docker.com/products/docker-desktop)
+- **Linux**: Follow the [official installation guide](https://docs.docker.com/engine/install/)
+
+To verify Docker is installed correctly, run:
+```bash
+docker --version
+```
+
+### Other Requirements
+- Wine dataset (if not included in the container)
+
+## Quick Start
+
+## Running the Analysis
+
+### Step 1: Clean the Project
```

Comment:
The user don't have the project yet. So, I think instead of this, we should ask use to run `docker pull` command to pull our project image from Dockerhub.